packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -401,45 +420,8 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   void acceptGesture(int pointer) {
     assert(!_acceptedActivePointers.contains(pointer));
     _acceptedActivePointers.add(pointer);
-    if (_state != _DragState.accepted) {
-      _state = _DragState.accepted;
-      final OffsetPair delta = _pendingDragOffset;
-      final Duration? timestamp = _lastPendingEventTimestamp;
-      final Matrix4? transform = _lastTransform;
-      final Offset localUpdateDelta;
-      switch (dragStartBehavior) {
-        case DragStartBehavior.start:
-          _initialPosition = _initialPosition + delta;
-          localUpdateDelta = Offset.zero;
-        case DragStartBehavior.down:
-          localUpdateDelta = _getDeltaForDetails(delta.local);
-      }
-      _pendingDragOffset = OffsetPair.zero;
-      _lastPendingEventTimestamp = null;
-      _lastTransform = null;
-      _checkStart(timestamp, pointer);
-      if (localUpdateDelta != Offset.zero && onUpdate != null) {
-        final Matrix4? localToGlobal = transform != null ? Matrix4.tryInvert(transform) : null;
-        final Offset correctedLocalPosition = _initialPosition.local + localUpdateDelta;
-        final Offset globalUpdateDelta = PointerEvent.transformDeltaViaPositions(
-          untransformedEndPosition: correctedLocalPosition,
-          untransformedDelta: localUpdateDelta,
-          transform: localToGlobal,
-        );
-        final OffsetPair updateDelta = OffsetPair(local: localUpdateDelta, global: globalUpdateDelta);
-        final OffsetPair correctedPosition = _initialPosition + updateDelta; // Only adds delta for down behaviour
-        _checkUpdate(
-          sourceTimeStamp: timestamp,
-          delta: localUpdateDelta,
-          primaryDelta: _getPrimaryValueFromOffset(localUpdateDelta),
-          globalPosition: correctedPosition.global,
-          localPosition: correctedPosition.local,
-        );
-      }
-      // This acceptGesture might have been called only for one pointer, instead
-      // of all pointers. Resolve all pointers to `accepted`. This won't cause
-      // infinite recursion because an accepted pointer won't be accepted again.
-      resolve(GestureDisposition.accepted);
+    if (onlyDispatchDragCallbacksWhenThresholdMet ? _hasDragThresholdBeenMet : !onlyDispatchDragCallbacksWhenThresholdMet) {
```

Comment:
Nit: Is this any easier to understand?

```suggestion
    if (onlyDispatchDragCallbacksWhenThresholdMet ? _hasDragThresholdBeenMet : true) {
```

Reply:
Changed to `!onlyAcceptDragOnThreshold || _hasThresholdBeenMet`